libs/sdk-ui-web-components/README.md

```diff
@@ -0,0 +1,130 @@
+# GoodData Web Component
+
+This library provides a set of Custom Elements (aka Web Components) to easy the embedding of GoodData visualizations
+into web applications.
+
+The library is meant to be hosted by GoodData.CN and GoodData Cloud for easier integration.
+At this time we do not provide an NPM package that you can import directly.
+
+## Integration
+
+To integrate the library into your app, all you need to do is to integrate a script tag with a correct URL to the `<head>`
+section of your application.
+
+```html
+<script type="module" src="https://{your-GD-server-URL}/components/{workspaceId}.js?auth=sso"></script>
+
+<!-- for example -->
+<script type="module" src="https://analytics.example.com/components/my-workspace.js?auth=sso"></script>
```

Comment:
Please use https://example.gooddata.com as the placeholder URL, it will link to a nice page explaining what happened in case someone actually tries to use this as is. See WEB-3288 for more details.

Reply:
This applies to several other places in this readme.